### DISCUSSION TITLE
网盘间能互传吗？？

### DISCUSSION THREAD

**Discussion by kitkw** [AlistGo/alist#4321] [Q&A]
E5 qyi续订翻车了，搞了个新的。alist能不能从一个onedrive传输给另一个onedrive??

**Comment by ririye** [comment]
我也4月5号翻车的，alist自带的网盘间复制功能对onedrive的兼容性有点差，而且关机重启以后不能继续任务，而且还经常出现复制失败的文件，一个号2t的东西用alist的复制功能搞了半个多月才搞定，第二个号里590多g的东西用了aria2离线下载的方法搞也搞了1周多，也是经常出现失败的情况，还是电脑重启不能继续，然后我怕剩下的几天不够搞最后几个号的东西啦作死尝试了已经被微软收购了的mover.io搞来试试，结果最后4个号共9t的东西3天就搞定了而且还不怕电脑重启，开始几天害怕像网上流言说的用mover.io的微软统统封号，提心吊胆了好久结果4月末用的新mover.io到现在微软还没封我，我分析以前的mover.io是私人制作的，支持各种网盘间互相复制，微软就不让用，现在微软给收购了就只支持微软自己的东西互相复制了，就让随便用啦，以上是个人经验，你要用的话网址给你，网址就是：mover.io

**Reply by kitkw** [reply]
> 我也4月5号翻车的，alist自带的网盘间复制功能对onedrive的兼容性有点差，而且关机重启以后不能继续任务，而且还经常出现复制失败的文件，一个号2t的东西用alist的复制功能搞了半个多月才搞定，第二个号里590多g的东西用了aria2离线下载的方法搞也搞了1周多，也是经常出现失败的情况，还是电脑重启不能继续，然后我怕剩下的几天不够搞最后几个号的东西啦作死尝试了已经被微软收购了的mover.io搞来试试，结果最后4个号共9t的东西3天就搞定了而且还不怕电脑重启，开始几天害怕像网上流言说的用mover.io的微软统统封号，提心吊胆了好久结果4月末用的新mover.io到现在微软还没封我，我分析以前的mover.io是私人制作的，支持各种网盘间互相复制，微软就不让用，现在微软给收购了就只支持微软自己的东西互相复制了，就让随便用啦，以上是个人经验，你要用的话网址给你，网址就是：mover.io

感谢回复！！我已经转移完毕了，也是用mover。差不多2天时间2.5T东西

**Reply by kitkw** [reply]
> 我也4月5号翻车的，alist自带的网盘间复制功能对onedrive的兼容性有点差，而且关机重启以后不能继续任务，而且还经常出现复制失败的文件，一个号2t的东西用alist的复制功能搞了半个多月才搞定，第二个号里590多g的东西用了aria2离线下载的方法搞也搞了1周多，也是经常出现失败的情况，还是电脑重启不能继续，然后我怕剩下的几天不够搞最后几个号的东西啦作死尝试了已经被微软收购了的mover.io搞来试试，结果最后4个号共9t的东西3天就搞定了而且还不怕电脑重启，开始几天害怕像网上流言说的用mover.io的微软统统封号，提心吊胆了好久结果4月末用的新mover.io到现在微软还没封我，我分析以前的mover.io是私人制作的，支持各种网盘间互相复制，微软就不让用，现在微软给收购了就只支持微软自己的东西互相复制了，就让随便用啦，以上是个人经验，你要用的话网址给你，网址就是：mover.io

现在有什么保活项目吗？

**Reply by ririye** [reply]
我现在用MS365 E5 Renew Plus桌面版登录调用一个月，换MS365 E5 Renew X网页版非登录调用一个月这样轮流来保活中

**Reply by kitkw** [reply]
这个需要一直开机打开软件…….

**Reply by ririye** [reply]
那你就一直用MS365 E5 Renew X网页版，这个不用一直开机

**Reply by cmj2002** [reply]
> 我现在用MS365 E5 Renew Plus桌面版登录调用一个月，换MS365 E5 Renew X网页版非登录调用一个月这样轮流来保活中

@ririye 现在怎么样，没被封吧？我续订失败快过期了想试试

**Reply by ririye** [reply]
一直存活到现在，续期了两次了都
